### ISSUE TITLE
IO Enhancement Request: readlines /line-seq

### ISSUE THREAD

**Issue by zachcp** [pixie-lang/pixie#180]
readlines/line-seq would be great additions to pixie.

I started penciling one out but run into issues: its all a bit new. Instead of using strings is there a way to recognize the byte as end of line?  

```[clojure]
(defn readline
  ([filestream]
    readline filestream [])
  ([filestream acc]
    (let [lineend (hash-set "\n" "\r")
          acc acc
          c (str (-read filestream))]
      (if (contains? lineend c)
       (apply string acc)
       (readline filestream (conj acc c))))))
```

**Comment by halgari** [comment]
You can use `char` to convert ints to characters, `nth` on a string to get a character from it, and `int` will convert a character to a integer. These might help.

**Comment by kgann** [comment]
@zachcp This is implemented in #193

**Comment by zachcp** [comment]
@kgann  nice!
